src/auth/strategies/at.strategy.ts

```diff
@@ -34,20 +34,24 @@ export class AtStrategy extends PassportStrategy(Strategy, "jwt-at") {
     }
 
     async validate(payload: any) {
-        const userInDb = await this.usersService.getUserRolesById(payload.sub);
+        //Get user roles
```

Comment:
Would it be helpful to annotate the return type of this validate method? It looks like it would be Promise<UserReq | undefined>

Reply:
Good idea, updated it